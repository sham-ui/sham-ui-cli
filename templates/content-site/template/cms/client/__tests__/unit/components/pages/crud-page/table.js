import PagesCrudPageTable  from '../../../../../src/components/pages/crud-page/table.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( PagesCrudPageTable, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
