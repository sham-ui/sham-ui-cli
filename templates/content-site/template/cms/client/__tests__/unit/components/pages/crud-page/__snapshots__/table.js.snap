// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "ID": "component",
    "confirmDeleteText": [Function],
    "deleteItem": [Function],
    "deleteModalVisible": false,
    "items": Array [],
    "onSelectItem": [Function],
    "refresh": [Function],
    "showErrors": [Function],
    "typeName": "",
  },
  "html": "
<!--0-->
<!--1-->
",
}
`;
